--- Z80Monitor.asm	2025-10-10 19:03:02.510592500 -0500
+++ Z80Monitor_c000.asm	2025-10-10 18:16:41.487947500 -0500
@@ -13,6 +13,9 @@
             INCLUDE "CONSTANTS.asm" ; copy or edit one of the 
                                   ; CONSTANTS-aaaa-pp.asm files to
                                   ; CONSTANTS.asm
+
+ROM_BOTTOM_c000	equ $c000
+
 SCAN:        EQU     005FEh
 
 ;ROM_BOTTOM:  EQU    0F000h		; Bottom address of ROM
@@ -24,7 +27,7 @@
 ;UART_BASE:  EQU     0E0h        ; Base port address, DART uses 4 ports
 
 
-            ORG ROM_BOTTOM
+            ORG ROM_BOTTOM_c000
 
 ROUTINES:
 R_MAIN:     JP      MAIN            ; init DART and starts command loop
@@ -36,9 +39,9 @@
             DEFS    3
             DEFS    3
             
-            ORG ROM_BOTTOM + 24     ; room for eight routine entries
+            ORG ROM_BOTTOM_c000 + 24     ; room for eight routine entries
 
-            ORG ROM_BOTTOM + $38     ; IRQ routine
+            ORG ROM_BOTTOM_c000 + $38     ; IRQ routine
 
 IRQ:		ex af,af'
 		exx
@@ -48,7 +51,7 @@
 		exx
 irq_exit:	reti
 
-            ORG ROM_BOTTOM + $66     ; nmi routine
+            ORG ROM_BOTTOM_c000 + $66     ; nmi routine
 
 NMI:		jp IRQTAB+(NMIV-IRQTABR)
 
@@ -61,6 +64,7 @@
 
 		include "eeprom_prog.asm";
 		INCLUDE "CFDriver.asm"
+;		INCLUDE	"DARTDriver.asm"
 		INCLUDE	"CONIO.asm"		; use UART for console
 ;		INCLUDE	"CONIO_SIO.asm"		; use SIO for console
 
@@ -238,7 +242,7 @@
         CALL    CLEAR_SCREEN        ;Clear the terminal screen
         LD      HL, MNMSG1          ;Print some messages
         CALL    CON_PRT_STR
-        LD      HL, ROM_BOTTOM
+        LD      HL, ROM_BOTTOM_c000
         CALL    CON_PRINTHWORD
         LD      HL, MNMSG3B         ; 2nd part, RAM
         CALL    CON_PRT_STR
@@ -360,11 +364,10 @@
 		CALL	CON_PRT_NL
 		ret
 
-;		INCLUDE	"DARTDriver.asm"
 		INCLUDE	"UARTDriver.asm"
 
 endprog	equ $
 
-	output_bin "z80monitor.bin",0,endprog    ; 
-	output_intel "z80monitor.hex",0,endprog    ;
-	output_list "z80monitor.lst"
+	output_bin "z80monitor_c000.bin",ROM_BOTTOM_c000,endprog-ROM_BOTTOM_c000    ; 
+	output_intel "z80monitor_c000.hex",ROM_BOTTOM_c000,endprog-ROM_BOTTOM_c000    ;
+	output_list "z80monitor_c000.lst"
